Fix:
./buf.c:136:7: error: implicitly declaring library function 'memcmp' with type 'int (const void *, const void *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
--- buf.c.orig	2001-03-15 02:32:59.000000000 -0600
+++ buf.c	2021-07-29 11:14:25.000000000 -0500
@@ -14,6 +14,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdarg.h>
+#include <string.h>
 #include <unistd.h>
 #include <ctype.h>
 #include <err.h>
